build_scripts/build_usd.py

```diff
@@ -1643,6 +1643,102 @@ def InstallAnimX(context, force, buildArgs):
 
 ANIMX = Dependency("AnimX", InstallAnimX, "include/animx.h")
 
+############################################################
+# Vulkan SDK components
+VULKAN_SDK_VERSION = "1.3.296.0"
+
+VULKAN_HEADERS_URL = f"https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/vulkan-sdk-{VULKAN_SDK_VERSION}.zip"
+def InstallVulkanHeaders(context, force, buildArgs):
+    with CurrentWorkingDirectory(DownloadURL(VULKAN_HEADERS_URL, context, force, destFileName=f'Vulkan-Headers-{VULKAN_SDK_VERSION}.zip')):
+        extraArgs = buildArgs + [
+            '-DVULKAN_HEADERS_ENABLE_INSTALL=ON',
+            '-DVULKAN_HEADERS_ENABLE_MODULE=OFF',
+            '-DVULKAN_HEADERS_ENABLE_TESTS=OFF',
+        ]
+        RunCMake(context, force, extraArgs)
+
+VULKAN_HEADERS = Dependency("Vulkan-Headers", InstallVulkanHeaders, "include/vulkan/vulkan.h")
+
+VULKAN_UTILITY_LIBRARIES_URL = f"https://github.com/KhronosGroup/Vulkan-Utility-Libraries/archive/refs/tags/vulkan-sdk-{VULKAN_SDK_VERSION}.zip"
+def InstallVulkanUtilityLibraries(context, force, buildArgs):
+    with CurrentWorkingDirectory(DownloadURL(VULKAN_UTILITY_LIBRARIES_URL, context, force, destFileName=f'Vulkan-Utility-Libraries-{VULKAN_SDK_VERSION}.zip')):
+        extraArgs = buildArgs + [
+            '-DBUILD_TESTS=OFF',
+        ]
+        RunCMake(context, force, extraArgs)
+
+VULKAN_UTILITY_LIBRARIES = Dependency("Vulkan-Utility-Libraries", InstallVulkanUtilityLibraries, "include/vulkan/vk_enum_string_helper.h")
+
+VULKAN_LOADER_URL = f"https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/vulkan-sdk-{VULKAN_SDK_VERSION}.zip"
+def InstallLoaderVulkan(context, force, buildArgs):
+    with CurrentWorkingDirectory(DownloadURL(VULKAN_LOADER_URL, context, force, destFileName=f'Vulkan-Loader-{VULKAN_SDK_VERSION}.zip')):
+        extraArgs = buildArgs + [
+            f'-DVULKAN_HEADERS_INSTALL_DIR={context.instDir}',
+            '-DBUILD_TESTS=OFF',
+            '-UPDATE_DEPS=ON',
+        ]
+        RunCMake(context, force, extraArgs)
+
+VULKAN_LOADER = Dependency("Vulkan-Loader", InstallLoaderVulkan, "lib/cmake/VulkanLoader/VulkanLoaderConfig.cmake")
+
+SPIRV_REFLECT_URL = f"https://github.com/KhronosGroup/SPIRV-Reflect/archive/refs/tags/vulkan-sdk-{VULKAN_SDK_VERSION}.zip"
+def InstallSpirvReflect(context, force, buildArgs):
+    with CurrentWorkingDirectory(DownloadURL(SPIRV_REFLECT_URL, context, force, destFileName=f'SPIRV-Reflect-{VULKAN_SDK_VERSION}.zip')):
+        extraArgs = buildArgs + [
+            '-DSPIRV_REFLECT_EXECUTABLE=OFF',
+            '-DSPIRV_REFLECT_STATIC_LIB=ON',
+            '-DSPIRV_REFLECT_BUILD_TESTS=OFF',
+            '-DSPIRV_REFLECT_ENABLE_ASSERTS=OFF',
+            '-DSPIRV_REFLECT_ENABLE_ASAN=OFF',
+        ]
+        RunCMake(context, force, extraArgs)
+        CopyFiles(context, 'spirv_reflect.h', 'include')
+        CopyDirectory(context, 'include/spirv', 'include/spirv')
```

Comment:
I think we should remove this as the spirv header file is not used and is not supposed to be used.

Reply:
This header is the one included by `spirv_reflect.h`. It's needed.

Reply:
Ah, you are right. Never mind. I somehow got the idea that the build script also installed the spirv headers. But it doesn't. Sorry for the noise.